### ISSUE TITLE
Новые комментарии не приходят на сайт

### ISSUE THREAD

**Issue by iprst** [audetv/fct-parser#24]
Сайт svodd.ru не обновляет комментарии. Последний комментарий темы тот же, что в телеграме, от 0:08 15.12.2024 (про два танкера). В приложении и в браузере картина одна и та же.

https://github.com/terratensor/svodd/issues/360

**Comment by audetv** [comment]
Спасибо. Восстановлено.

**Comment by audetv** [comment]
Точно не понял, что это было. В итоге физически перезапустил сервер (ноду) на котором запускаются службы, и заработало. Не отрабатывал крон, не приходили команды на сервер на запуск контейнеров для парсера и для обновления. Команды уходили т.к. в логах крона все было ок и без ошибок, но последний контейнер на ноде в тот момент был 17 часов назад запущен. В общем перезагрузка помогла.

Связываю с тем, что у провайдера с пятницы на субботу проходил работы на сетевом оборудовании, меняли архитектуру сети в очередной раз. И утром в субботу сайт не работал, сервер не отвечал по ssh, работал странно, явно были какие то сетевые проблемы. Я написал тикет, в итоге уже через пол часа все запустили. И вот после этого в воскресение отвалилась служба крон. Полагаю это продолжение последствий работ в сети. Так что держим под наблюдением и если что-то отвалится просьба подавать сигнал.

**Comment by iprst** [comment]
Вновь перестали проходить комментарии через парсер — сайт и телега не обновляются. 
Номер 481965 не приходит.

**Comment by audetv** [comment]
Да, вижу, спасибо. Не понятно, ошибок нигде не вижу, главная нода показывает, что запускает все по палну, ошибок нет, а на worker'e не происходит запуск. Немного поразбираюсь и перезагружу сервер, не нравится мне это.

**Comment by audetv** [comment]
Выключил все серверы, запустил последовательно все структуру снова, обновил пакеты на серверах. Продолжим наблюдение. Если поведение будет продолжаться, в планах перенос всех сервисов с этой ноды, на которой зависает, на новую.
